apps/sms/views/conversation/js/compose.js

```diff
@@ -1,7 +1,7 @@
 /* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
 
-/*global Settings, Utils, Attachment, MozActivity,
```

Comment:
__issue:__ I see ```JavaScript Error: "TypeError: dom.message is null" {file: "app://sms.gaiamobile.org/views/conversation/js/compose.js" line: 496}``` in logcat if I share image via Messages app with your PR rebased on the latest master - could you please take a look?